advocacy_docs/partner_docs/VeritasGuide/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Veritas Implementation Guide'
+indexCards: simple
+directoryDefaults:
+  iconName: handshake
+---
+
+<p align="center">
+  <img  src="Images/EDBPartnerProgram.png">
```

Comment:
Might be just the preview page but I see a black boarder around our logo.  We need to make sure that does not show up in final version.

Reply:
This is copied from the template - we need to check and fix the template

Reply:
Image fixed in template and guide